src/libraries/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -235,20 +237,20 @@ internal static bool TryFindHierarchyMount(string mountInfoFilePath, string subs
         /// <param name="subsystem">The subsystem, e.g. "memory".</param>
         /// <param name="path">The found path, or null if it couldn't be found.</param>
         /// <returns></returns>
-        private static bool TryFindCGroupPathForSubsystem(string subsystem, out string path)
+        private static bool TryFindCGroupPathForSubsystem(string subsystem, out string? path)
```

Comment:
Should this and the below function have `[NotNullWhen(true)]` on the `out string? path` parameter?